### DISCUSSION TITLE
k32 vs k35.

### DISCUSSION THREAD

**Discussion by chiaXXL** [Chia-Network/chia-blockchain#3465] [General]
I didnt find any information with clear answer: does the k35 will ok for farming?
Is the any difference for chance to win? One 800 Gb file vs 8 by 100 gb?

**Comment by dorkmo** [comment]
chance to win is proportional to the file size. file size is proportional to time to create. so no difference or advantage. k32 will be retired first, then at some point only k33 will be accepted.

**Comment by moshenskyDV** [comment]
That's answer to your question by plain language: [https://www.youtube.com/watch?v=knNPLoX5Jto](https://www.youtube.com/watch?v=knNPLoX5Jto)
